kvdb/interface.go

```diff
@@ -109,6 +109,10 @@ type ExtendedRBucket interface {
 
 	// Prefetch will attempt to prefetch all values under a path.
 	Prefetch(paths ...[]string)
+
+	// ForAll is an optimized version of ForEach with the limitation that no
```

Comment:
To not break existing etcd `ExtendedBucket` impl you need to add a `ForAll` implementation there too. (otherwise the the cast  in `kvdb.Prefetch` won't succeed).

Reply:
Added